### PR TITLE
Fix default value of input, remove now unused allowlist entry for epoll

### LINKED ISSUE #9741: Daily tests failed on Thu Feb 16 2023
Runs listed here: https://github.com/python/typeshed/actions/workflows/daily.yml

### PR DESCRIPTION
**Pull request by hauntsaninja**
Fixes #9741

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]
Confirmed by reading the code.

**Comment by JelleZijlstra** [comment]
Though it seems like a few stubtests haven't picked up the runtime change yet.

**Comment by hauntsaninja** [comment]
Okay, I was confused but I wasn't reading logs right. The 3.11 failures are from an unused allowlist entry caused by my `inspect.signature` fixes

**Comment by JelleZijlstra** [comment]
Lesson: stop fixing bugs.

**Review by JelleZijlstra — APPROVED** [review]

File: .github/workflows/stubtest_stdlib.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       matrix:
         # tkinter doesn't import on macOS 12
         os: ["ubuntu-latest", "windows-latest", "macos-11"]
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.7", "3.8", "3.9", "3.10.10", "3.11.2"]  # TODO: unpin micro versions
```
Comment (Akuli) [inline .github/workflows/stubtest_stdlib.yml:35]: This list of versions should be copied to `.github/workflows/daily.yml`.

**Review by Akuli — APPROVED** [review]

**Pull request merged**